test/unit/orgTest.ts

```diff
@@ -710,12 +729,40 @@ describe('Org Tests', () => {
 
   describe('sandbox org config', () => {
     it('set field', async () => {
+      // Stub exists so only the auth file and sandbox config file exist. No users config file.
+      stubMethod($$.SANDBOX, ConfigFile.prototype, 'exists').callsFake(async function() {
```

Comment:
The delete should a different test.  The original test just handled verifying the set sandbox org config worked.